docs/source/building_an_experiment.rst

```diff
@@ -191,9 +142,9 @@ ExperimentPrototype class.
 
 This means the ExperimentPrototype class must be imported at the start of the experiment file ::
 
-    from experiments.experiment_prototype import ExperimentPrototype
+    from experiment_prototype.experiment_prototype import ExperimentPrototype
 
-Please name the class within the experiment file in a similar fashion to the file as the class name
+Please name the class within the experiment file in a similar fashion to the file, as the class name
 is written to the datasets produced.
 
 The experiment has the following experiment-wide attributes:
```

Comment:
Are these attributes able to be imported from experiment_prototype.py?

Reply:
I decided not to, since the text written here is more verbose and gives more detail than in the docstring of ExperimentPrototype. I also don't expect these parameters to change as often, as opposed to ExperimentSlice which is more malleable.